Watch Extension/HomeView.swift

```diff
@@ -8,18 +8,20 @@ import SwiftUI
    behaves well from a UI standpoint when using the watch
  - App icon
  - Think about subscriptions or IAP for this
- - Add unidirectional flow architecture for better state management
  - Allow stopping current feeding
  - Allow pausing current feeding
+ - Load all feedings when starting up?
+ - Make sure iPhone is available and logged in
  */
 
 struct HomeView: View {
+    @ObservedObject var store: Store<AppState, AppAction> = Store(initialValue: AppState(), reducer: appReducer)
     @State private var isShowingSheet = false
     var body: some View {
         // TODO: think more about the layout of these two
         VStack {
-            // TODO: fill this in with active feedings
-            List(1 ... 10, id: \.self) { _ in
+            List(store.value.activeFeedings, id: \.self) { feeding in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/443d289c50906f54d351235f32088d9c382abf56/Watch Extension/HomeView.swift#L23"></span>Unused parameter “feeding” in a closure should be replaced with _.<br />
<code>unused_closure_parameter</code> <code>HomeView.swift:23</code></td>
    </tr>
  </tbody>
</table>